Makefile

```diff
@@ -145,7 +146,7 @@ ifeq ($(TARGET_PLATFORM),)
 	endif
 endif
 
-BUILD_INDOCKER_ARG := --env IN_DOCKER=1 --volume $(ROOT):/mnt --user $(shell id -u):$(shell id -g) --platform=linux/$(TARGET_PLATFORM)
+BUILD_INDOCKER_ARG := --env IN_DOCKER=1 --env HTTP_PROXY=${HTTP_PROXY} --env HTTPS_PROXY=${HTTPS_PROXY} --env GOPROXY=${GOPROXY} --volume $(ROOT):/mnt --user $(shell id -u):$(shell id -g) --platform=linux/$(TARGET_PLATFORM)
```

Comment:
Do we really need `HTTP_PROXY` and `HTTPS_PROXY`? I think it should not be considered by us.

Reply:
`HTTP_PROXY` and  `HTTPS_PROXY` are the most common solution to access web resources that cannot be accessed by the local networks. 
For our use cases, we need  `HTTPS_PROXY` to download binaries from the GitHub release.

Reply:
I mean the network connectivity should not consider by the project itself. 🙈

`GOPROXY` makes sense because it's a part of `go env`, and actually I prefer to use something like `LD_FLAGS` for passing all the configurable flags or ENVs into go command.

Reply:
Building in containers makes it difficult to pass HTTP proxy configurations, so I think we have the duty to make it easier (as router proxies or transparent proxies are more difficult to set up).

Reply:
> For our use cases, we need HTTPS_PROXY to download binaries from the GitHub release.

It seems we only need to download binaries when we build dockerimage. And HTTP_PROXY/HTTPS_PROXY have already been set in `DOCKER_BUILD_ARGS` at line 7. Do I miss something?

Reply:
It seems you are right, we do not need to download binaries in this step. Let me clear my opinion, if we develop and build binaries on the host machine, the network connectivity should not consider by the project itself. However, now all the dev tools and build tools are running in containers, so we have the duty to provide ways to configure HTTP proxies in the container.

For pure golang projects, env `HTTPS_PROXY` and `GOPROXY` both works, and developers can choose one of them to solve network problems.

Reply:
> > For our use cases, we need HTTPS_PROXY to download binaries from the GitHub release.
> 
> It seems we only need to download binaries when we build dockerimage. And HTTP_PROXY/HTTPS_PROXY have already been set in `DOCKER_BUILD_ARGS` at line 7. Do I miss something?

A lot of dependencies are downloaded in the container while we are building the binary inside the container.

Reply:
> A lot of dependencies are downloaded in the container while we are building the binary inside the container.

I do not get it, I think the `dev-env` already contains the binary tools so we do not need to download them anymore. 🤔

Reply:
> It seems you are right, we do not need to download binaries in this step. Let me clear my opinion, if we develop and build binaries on the host machine, the network connectivity should not consider by the project itself. However, now all the dev tools and build tools are running in containers, so we have the duty to provide ways to configure HTTP proxies in the container.
> 
> For pure golang projects, env `HTTPS_PROXY` and `GOPROXY` both works, and developers can choose one of them to solve network problems.


I get your point. LGTM!